src/annotator/highlighter.js

```diff
@@ -259,7 +259,8 @@ export function highlightRange(range, cssClass = 'hypothesis-highlight') {
     const highlightEl = document.createElement('hypothesis-highlight');
     highlightEl.className = cssClass;
 
-    nodes[0].parentNode.replaceChild(highlightEl, nodes[0]);
+    const parent = /** @type {Node} */ (nodes[0].parentNode);
+    parent.replaceChild(highlightEl, nodes[0]);
```

Comment:
`nodes` is now typed, so TS complains that `parentNode` may be null. We know that in this context it isn't.